css/css-cascade/all-prop-revert-layer.html

```diff
@@ -345,8 +355,10 @@
   user-select: all;
   vector-effect: non-scaling-stroke;
   vertical-align: 123px;
+  view-timeline: --foo inline 10px;
+  view-transition-name: --foo;
   visibility: collapse;
-  white-space-collapse: preserve;
+  white-space: pre;
```

Comment:
The test checks longhands, but it's fine to write shorthands here and let them expand.
In fact, most browsers support `white-space` but not the longhands, so 20fd15255b8ed5274839be72406bd5c5b7500af9 shouldn't have removed `white-space`.
It would be fine to leave the longhands, but no need (except for `white-space-trim`, since all widely supported values of the shorthand expand to the initial `none`).